bedrock/firefox/templates/firefox/family/includes/macros.html

```diff
@@ -8,8 +8,10 @@
   title,
   anchor
 ) -%}
-<a class="c-module-tag" href="#{{ anchor }}">
+<a class="c-module-tag" href="#{{ anchor }}" tabindex="0">
   <h2 class="c-module-tag-title">{{ title }}</h2>
+  <span class="c-module-copy" role="tooltip">Copy to clipboard</span>
+  <span class="c-module-copied" role="tooltip" aria-live="polite">Copied to clipboard!</span>
```

Comment:
I think this is the right idea, because content will appear dynamically that we want a screenreader to notice, but because this element is `display:none` it is removed from the accessibility tree and I'm not sure it will be registered as a live region... my understanding is that you need to place this attribute on a containing element that always exists on the page, then changes within it can be noticed (but I haven't tested yet)

Reply:
tested with safari/voiceover, seems to be working as-is 👍